verify/src/metadata_schema.json

```
@@ -123,6 +124,30 @@
         },
         "perpendicular_baseline": {
           "type": "number"
+        },
+        "version": {
+          "type": "string"
+        },
+        "weather_model": {
+          "type": "array",
```

Discussion in slack settled on not including this attribute at all when no weather model was used, so we can enforce that the list is non-empty when a weather model *is* used.
```suggestion
          "type": "array",
          "minItems": 1,
```